playground/Playground.ServiceDefaults/Extensions.cs

```diff
@@ -83,9 +83,12 @@ private static IHostApplicationBuilder AddOpenTelemetryExporters(this IHostAppli
         // builder.Services.AddOpenTelemetry()
         //    .WithMetrics(metrics => metrics.AddPrometheusExporter());
 
-        // Uncomment the following lines to enable the Azure Monitor exporter (requires the Azure.Monitor.OpenTelemetry.Exporter package)
-        // builder.Services.AddOpenTelemetry()
-        //    .UseAzureMonitor();
+        // Uncomment the following lines to enable the Azure Monitor exporter (requires the Azure.Monitor.OpenTelemetry.AspNetCore package)
+        //if (!string.IsNullOrEmpty(builder.Configuration["APPLICATIONINSIGHTS_CONNECTION_STRING"]))
```

Comment:
Is this really the best environment variable to hard code into our templates? It doesn't seem like it will last.

Reply:
It's literally what the Azure SDK looks for. I think it'll last.